pkg/apis/core/validation/cloudprofile_test.go

```diff
@@ -1455,4 +1509,185 @@ var _ = Describe("CloudProfile Validation Tests ", func() {
 			})
 		})
 	})
+
+},
+	Entry("CloudProfileCapabilities = false", false),
+	Entry("CloudProfileCapabilities = true", true),
+)
+var _ = Describe("CloudProfile with capabilities specific features", func() {
```

Comment:
Test for rejecting a CloudProfile with capabilities in it when the FeatureGate is off?